Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM python:3.10.9-slim
+
+# Install Postgres as pre requisite
+RUN apt-get update && \
+    apt-get install -y build-essential && \
+    apt-get install -y git && \
```

Comment:
What is git used for in the container?

Reply:
I probably don't need it I just copied how I installed postgres in Dockerfile.ci, I'll take that out